File: primerTrimestre/java basico/1secuenciales/Ej7sec.java
/**Realiza un programa que reciba una cantidad de minutos y muestre por pantalla a cuantas horas y minutos corresponde.
 *
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej7sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    int minutos;
    int minutosRes;
    int horasRes;
    
    //Introducimos la cantidad de minutos.
    System.out.println("Introduce una cantidad de minutos:");
    minutos = s.nextInt();
    
    //Cálculos
    horasRes = minutos/60; //Parte entera de los minutos para sacar las horas restantes.
    minutosRes = minutos%60; //Resto de los minutos
    
    //Imprime las horas y los minutos totales.    
    System.out.println(horasRes+" horas y "+minutosRes+" minutos.");
   
    
    
    
  }

}

File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej7sec.py
# Programa: ej7sec.py
# Propósito: Realiza un programa que reciba una cantidad de minutos y muestre por pantalla a cuantas horas y minutos corresponde.
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * num1
#   * num2
#   * num3
#
# Algoritmo:
#   LEER num1, num2, num3
#   media <-- (num1+num2+num3)/3
#   ESCRIBIR media


# Petición de datos
minutos = int(input("Inserte una cantidad de minutos: "))
minutos_res = int()
horas_res = int()
print("-------------------------------------------")

#Cálculos
horas_res = minutos//60
minutos_res = minutos%60

# Salida
print(horas_res, "horas y", minutos_res, "minutos.")



	
	

	






	
	

	
